core/globals.h

```diff
@@ -312,11 +312,22 @@ typedef struct _dr_stats_t {
  * Error codes of DR API routines.
  */
 typedef enum {
-    DR_ERROR_NONE,
-    DR_ERROR_INVALID_PARAM,
-    DR_ERROR_INSUFFICIENT_SPACE,
-    DR_ERROR_APP_ENCODING,
-    DR_ERROR_NOT_IMPLEMENTED,
+    /**
+     * Invalid parameter passed to the API routine.
```

Comment:
(Note that these short comments could be the `/**<  */` after-field same-line style.  Not requesting any change: just in case you weren't aware.)

Reply:
Yep, just that personally I prefer this style because the text looks nicer. With same-line style one will end up with a mix of single line and multi-line comments depending on the comments' lengths. Going with multi-line immediately maintains consistency :)

Reply:
OK sure